backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -43,6 +43,7 @@ public static IQueryable<Entity.Project> GenerateQuery(this PimsContext context,
                 .Include(p => p.TierLevel)
                 .Include(p => p.Risk)
                 .Include(p => p.Agency)
+                .Include(p => p.Workflow)
```

Comment:
So the list view is able to access this information to determine if delete icon will be displayed based off of `workflowCode`